pyproject.toml

```diff
@@ -12,8 +17,17 @@ edgedb = "^2.2.0"
 uvicorn = "^0.32.1"
 mypy = "^1.13.0"
 httpx = "^0.28.1"
+pyjwt = "^2.10.1"
+htmy = "^0.4.2"
+fasthx = {extras = ["htmy"], version = "^2.1.1"}
+logging = "^0.4.9.6"
```

Comment:
```suggestion
```

`logging` is a builtin module, the 12-years-old library is probably wrong?

Reply:
ha! thanks for catching that. I don't remember why I ended up installing this? maybe i had some tooling issue on my end.